tests/unit/snippets/test_files.py

```diff
@@ -16,6 +16,18 @@ def test_directory_instantiation(self):
         directory = DirectoryObject(self.directory)
         self.assertEqual(directory.path, self.directory.path)
 
+    def test_file_instantiation(self):
```

Comment:
Tests need to be extended with creation from absolute, including a conflict failure if we want to introduce one.